app/server.js

```diff
@@ -1,32 +1,35 @@
 import botkit from 'botkit'
-
-let channelMessages = []
+import schedule from 'node-schedule'
+import data from '../mock_data/milestones' 
+import moment from 'moment'
+import * as markdown from './markdown.js' 
 
 // botkit controller
 const controller = botkit.slackbot({
-  debug: false,
-})
+  debug: false, 
+}) 
+
+let channelMessages = []
 
 // initialize slackbot
 const slackbot = controller.spawn({
-  token: process.env.TASK_BOT_TOKEN,
+  token: process.env.SLACK_BOT_TOKEN,
```

Comment:
You changed your environment name to `SLACK_BOT_TOKEN`. It should be `TASK_BOT_TOKEN`